testsuites/CBLTester/CBL_Functional_tests/APITests/test_collection.py

```diff
@@ -0,0 +1,264 @@
+        scope = self.collection_obj.collectionScope(created_Collection)
+        retrivedName = self.scope_obj.scopeName(scope)
+        assert retrivedName == scopeName, "Incorrect Scope Object"
+
+    @pytest.mark.parametrize("db_name, collectionName, docId", [
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+    ])
+    def test_getDocument(self, db_name, collectionName, docId):
+        db = self.db_obj.create(db_name)
+        scopeName = '_default'
+        created_collection = self.db_obj.createCollection(db, collectionName, scopeName)
+        doc = self.doc_obj.create(docId)
+        self.collection_obj.saveDocument(created_collection, doc)
+        retrived_doc = self.collection_obj.getDocument(created_collection, docId)
+        retrived_docId = self.doc_obj.getId(retrived_doc)
+        assert docId == retrived_docId, "Document retrival failed"
+
+    @pytest.mark.parametrize("db_name, collectionName, docId", [
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+    ])
+    def test_deleteDocument(self, db_name, collectionName, docId):
+        db = self.db_obj.create(db_name)
+        scopeName = '_default'
+        created_collection = self.db_obj.createCollection(db, collectionName, scopeName)
+        doc = self.doc_obj.create(docId)
+        self.collection_obj.saveDocument(created_collection, doc)
+        retrived_doc = self.collection_obj.getDocument(created_collection, docId)
+        assert self.collection_obj.deleteDocument(created_collection, retrived_doc) == True
+
+    @pytest.mark.parametrize("db_name, collectionName, docId", [
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+    ])
+    def test_purgeDocument(self, db_name, collectionName, docId):
+        db = self.db_obj.create(db_name)
+        scopeName = '_default'
+        created_collection = self.db_obj.createCollection(db, collectionName, scopeName)
+        doc = self.doc_obj.create(docId)
+        self.collection_obj.saveDocument(created_collection, doc)
+        retrived_doc = self.collection_obj.getDocument(created_collection, docId)
+        assert self.collection_obj.purgeDocument(created_collection, retrived_doc) == True
+
+    @pytest.mark.parametrize("db_name, collectionName, docId", [
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+        (random_string(6), random_string(6), random_string(6)),
+    ])
+    def test_purgeDocumentById(self, db_name, collectionName, docId):
```

Comment:
Do we need to test this scenario? If we do, maybe this can be done in unit testing, since these system tests are expensive?